DevoxxClientCommon/src/main/java/com/devoxx/util/DevoxxSettings.java

```diff
@@ -117,6 +117,7 @@
     public static final String BADGE_SPONSOR = "badge-sponsor";
     public static final String RELOAD = "reload";
     public static final String RATING = "rating";
+    public static final String NEW_VERSION = "newVersion";
```

Comment:
It is probably best to give this a more explicit name like "version_3_0_0"?

Reply:
Good idea. Makes it easy to trace back.